### DISCUSSION TITLE
how can i open yazi with a keybind? (hyprland)

### DISCUSSION THREAD

**Discussion by xienuss** [sxyazi/yazi#705] [1. Q&A] [answered]
if I write `$filemanager = Yazi` in my config it just won't open, is there a way to exec kitty with yazi already open?

**Comment by sxyazi** [comment, marked as answer]
kitty:

```bash
kitty yazi
```

foot:

```bash
foot -e yazi
```

WezTerm:

```bash
wezterm -e yazi
```

I haven't installed other terminals, please refer to their documentation.

**Reply by xienuss** [reply]
thanks!
